nbs/core.ipynb

```diff
@@ -40,7 +40,8 @@
     "from typing import Callable, Dict, List, Optional\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #89.                   raise Exception('Check <code>Y_hat_df</code>, <code>Y_df</code> columns difference')
</pre><p>If this is still a todo, it would be helpful to have a clearer error message. Maybe something like: <code>It is currently excepted that the Y_hat_df and Y_df have the same columns.</code>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/hierarchicalforecast/pull/135/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
I changed the Exception in the next PR, to aprovechar current CI tests.
They are taking a lot of time in MacOS.